### ISSUE TITLE
Layer Panel - Colour Contrast Accessibility Improvement

### ISSUE THREAD

**Issue by jauyong** [GoogleForCreators/web-stories-wp#1217]
## Bug Description

Light elements should not have drop shadow in layer panel, and dark elements should receive a white border

## Screenshots

![image](https://user-images.githubusercontent.com/5874851/79270431-2d4fe580-7e6c-11ea-8fbf-14afc70c9b38.png)

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by samitron7** [comment]
Let's apply a 1 border radius (#fff, 50%)  to all shapes

**Comment by o-fernandez** [comment]
I will close this out as I believe all of this should be addressed with the new design/visual pass.
